syntax = "proto2";

import "nanopb.proto";

service Zone3 {
    rpc EccSign (EccSignRequest) returns (EccSignResponse) {}
}

message EccSignRequest {
    required bytes data = 1 [(nanopb).max_size = 256];
}

message EccSignResponse {
    required int32 result = 1;
    required bytes data = 2 [(nanopb).max_size = 256];
}
